EIPS/eip-4721.md

```diff
@@ -0,0 +1,32 @@
+---
+eip: 4721
+title: Transaction Expiration
+description: Proposes a transaction expiration block for every transaction.
+author: Lucas Vinzon (@SnowPrimate)
+discussions-to: https://ethereum-magicians.org/t/eip-4721-transaction-expiration/8252
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-01-25
+---
+
+## Abstract
+When in Ethereum mempool, every transaction should expire after a certain amount of blocks resulting in no cost to its users. 15.000 blocks would effectively have every transaction on the mempool for two days.
+
+## Motivation
+Gas cost, usability and serving its purpose for its users. Prevents waste of resources and benefits network usage.
+
+## Specification
+Once in the Mempool ALL transactions MUST expire after a certain amount of blocks. This applies to ALL transactions. It is OPTIONAL to lower the amount of blocks it will be live on the Mempool. It SHALL NOT result in gas cost to users.
```

Comment:
This needs to be more specific.  If this is a Core EIP then it means it is a consensus rule, but the specification as currently written gives no way to apply consensus validation for enforcement of this.